app/src/main/java/util/Validation.kt

```diff
@@ -0,0 +1,5 @@
+package util
+
+fun isValidInput(text: String): Boolean {
+    return text.replace("\\s".toRegex(), "") != ""
```

Comment:
What are you trying to achieve with this validation?
From my understanding, you are removing whitespace & checking that it is not empty which you have a method for why not use .filterNot { it.isWhitespace() }.isNotEmpty()

In addition you can create it as an extension function for String or even for TextView and remove the use of utils.